python/feathub/feature_views/transforms/sliding_window_transform.py

```diff
@@ -58,6 +64,19 @@ def __init__(
         self.filter_expr = filter_expr
         self.limit = limit
 
+        # TODO: Support infinite window size with non-zero step size.
+        if window_size == timedelta(0) and step_size > timedelta(0):
```

Comment:
Would it be better to use `step_size != timedelta(0)` for consistency with the check below?